src/printing.jl

```diff
@@ -10,6 +10,13 @@ function show_option(io::IO, m::MIME"text/plain", x)
     indent_print(io, ")")
 end
 
+function show_option(io::IO, m::MIME"text/html", x)
+    buf = IOBuffer()
+    show(buf, MIME("text/plain"), x)
+    printer = HTMLPrinter(buf; root_class="configurations-option-type")
+    show(io, m, printer)
```

Comment:
you can add:
```julia
write(io, """
<link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/JuliaDocs/ANSIColoredPrinters.jl@0.0.1/docs/src/assets/default.css" />
""")
```

Reply:
ah thanks, now it works, but with an ugly `Main.workspace` prefix tho.